### DISCUSSION TITLE
Custom hitbox handler

### DISCUSSION THREAD

**Discussion by Obyvante** [Minestom/Minestom#1579] [Help] [answered]
I plan to have a "zone" that handles the player's right-clicking using a custom hit box. How should I do it with Minestom? Should I custom code and check every right-click or do we have an API already for it?

For example, a 3D entity that has hit box 10,10,10 as a vector(simple example).

**Comment by KrystilizeNevaDies** [comment, marked as answer]
Your only solution is to manually check.
Minestom does not handle cases where the hotbox is larger than the client's hitbox.
